internal/parser.go

```diff
@@ -1789,10 +1789,9 @@ func inTableIM(p *parser) bool {
 		p.im = inExpressionIM
 		return true
 	case StartTagToken:
-		if isComponent(p.tok.Data) {
+		if isComponent(p.tok.Data) || isFragment(p.tok.Data) {
 			p.originalIM = inTableIM
 			p.im = inLiteralIM
-			p.exitLiteralIM = getExitLiteralFunc(p)
```

Comment:
I'm not sure why but removing this live fixes the weird white space issue I had in [`ea2dce3` (#926)](https://github.com/withastro/compiler/pull/926/commits/ea2dce32c80e682faa7c4de61ee6e36b7d1ccb86#diff-d0dbf4b902747f2cdc45290c88eb59b04d03bebcb2bd8872a0b49284a9dac9d1R2105)

Reply:
Hmm, this was definitely a hacky solution when I introduced it. I'm a little worried this will cause some other regressions, but I'm willing to try it.